src/main/ipc.js

```diff
@@ -58,6 +58,10 @@ function init (cwd) {
     e.returnValue = ret
   })
 
+  ipc.on('dispatchSyncNoRender', (e, ...args) => {
+    e.returnValue = dispatch(...args)
+  })
```

Comment:
Needed a separate message to be able to get data from the backend _without_ rendering automatically, i.e. get data outside of the normal state pipeline.